src/libraries/Microsoft.Extensions.Caching.Memory/ref/Microsoft.Extensions.Caching.Memory.cs

```diff
@@ -42,6 +42,8 @@ public partial class MemoryCacheOptions : Microsoft.Extensions.Options.IOptions<
         public MemoryCacheOptions() { }
         public Microsoft.Extensions.Internal.ISystemClock? Clock { get { throw null; } set { } }
         public double CompactionPercentage { get { throw null; } set { } }
+        [System.Obsolete("This is obsolete and will be removed in a future version.")]
```

Comment:
Is this just the default obsolete message or do we actually mean the fact that we want to remove in the future?

Reply:
This was the message that was on these members before they were removed.